group.go

```diff
@@ -268,7 +273,8 @@ func (g *Group) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 	switch msg := msg.(type) {
 	case tea.WindowSizeMsg:
-		g.WithHeight(max(g.height, min(g.fullHeight(), msg.Height-1)))
+		g.WithHeight(min(g.fullHeight(), msg.Height-helpMenuHeight))
+		g.WithWidth(msg.Width)
```

Comment:
This is what's breaking grid layout. Changes seem to work fine without it, so can remove it